R/geom_venn.R

```diff
@@ -21,10 +21,10 @@
 #'
 #' # use data.frame as input
 #' d <- tibble(value   = c(1, 2, 3, 5, 6, 7, 8, 9, 10, 12, 13),
-#'             `Set 1` = c(T, F, T, T, F, T, F, T, F,  F,  F),
-#'             `Set 2` = c(T, F, F, T, F, F, F, T, F,  F,  T),
-#'             `Set 3` = c(T, T, F, F, F, F, T, T, F,  F,  F),
-#'             `Set 4` = c(F, F, F, F, T, T, F, F, T,  T,  F))
+#'             `Set 1` = c(TRUE, FALSE, TRUE, TRUE, FALSE, TRUE, FALSE, TRUE, FALSE,  FALSE,  FALSE),
```

Comment:
Thank you very much to point out my bad writing. TRUE/FALSE is much better than T/F since T/F is not reserved keywords in R.